services/traefik/Makefile

```diff
@@ -17,70 +17,43 @@ SHELL := $(if $(IS_WIN),powershell.exe,/bin/bash)
 
 # Internal VARIABLES ------------------------------------------------
 # STACK_NAME defaults to name of the current directory. Should not to be changed if you follow GitOps operating procedures.
-STACK_NAME = $(notdir $(PWD))
+STACK_NAME = $(notdir $(shell pwd))
 SWARM_HOSTS = $(shell docker node ls --format={{.Hostname}} 2>/dev/null)
 TEMP_COMPOSE=.stack.${STACK_NAME}.yaml
 # Host machine IP
 export MACHINE_IP = $(shell source $(realpath $(CURDIR)/../../scripts/portable.sh) && get_this_ip)
-# Network that includes all services to monitor
-#  - by default it will create an overal attachable network called monitored_network
-ifeq ($(monitored_network),)
-MONITORED_NETWORK = monitored_network
-else
-MONITORED_NETWORK := $(monitored_network)
-endif
-export MONITORED_NETWORK
-
-# Network from which services are reverse-proxied
-#  - by default it will create an overal attachable network called public_network
-ifeq ($(public_network),)
-PUBLIC_NETWORK = public-network
-else
-PUBLIC_NETWORK := $(public_network)
-endif
-export PUBLIC_NETWORK
+
 
 # External VARIABLES
 include .env
 
 # TARGETS --------------------------------------------------
 include $(realpath $(CURDIR)/../../scripts/common.mk)
 
+# Makefile including function used by every services
+include ../common.Makefile
+
 .PHONY: up
 up-local: .init .create_secrets .env ${TEMP_COMPOSE}-local ## Deploys the stack using provided certificates
 	docker stack deploy -c ${TEMP_COMPOSE}-local ${STACK_NAME}
 
+.PHONY: up-aws
+up-aws: .env ${TEMP_COMPOSE}-aws ## Deploys the stack aws 
+	@echo "test"
```

Comment:
MINOR: remove these echos
Use instead inforative comments, eg.
`` # deploying ${STACK_NAME} to aws ``